backend/src/main/java/de/laura/project/service/AnswerRandomizerService.java

```diff
@@ -0,0 +1,21 @@
+package de.laura.project.service;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.stereotype.Service;
+
+
+@Data
+@AllArgsConstructor
+@Service
+
+public class AnswerRandomizerService {
+
+
+    public int generateRandomNumber() {
+
+        int number = (int) (Math.random() * 4.0);
+
+        return number;
+    }
```

Comment:
```suggestion

  private static final int NUMBER_OF_ANSWERS = 4;

    public int generateRandomNumber() {

        return (int) (Math.random() * NUMBER_OF_ANSWERS);
    }
```

otherwise the number 4 is a little bit magic :D

Reply:
Good point.